### PR TITLE
Forms Automated Test - LPS-132832 | master

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase [hunk 1]
```diff
@@ -76,13 +76,46 @@ definition {
...
+
+		FormPortlet.submitWithValidationError(validationErrorMessage = "This field is required.");
+
+		FormFields.editText(
+			fieldName = "Numeric",
+			fieldValue = "123");
+
+		FormPortlet.submitSuccessfully();
+
+
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase]: Great job @leticiachiu!

The only thing missing was the `Assert the Success Page' step.

```
FormPortlet.viewSuccessPage(
	successPageDescription = "Your information was successfully received. Thank you for filling out the form.",
	successPageTitle = "Thank you.");
```

Can you please add this and resolve the conflicts?

Thanks!

Comment (leticiachiu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase]: hello, carol! this macro actually does the assert of success too:
  `macro submitSuccessfully {
		FormPortlet.submit();

		WaitForElementPresent(locator1 = "Message#SUCCESS_DISMISSIBLE");

		Alert.viewSuccessMessage();
	}`

Comment (carolmariaabb) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase]: Thanks for the explanation, @leticiachiu ! Can you resolve the conflicts so I can forward the PR? Thanks!

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**